packages/features/webhooks/lib/factory/WebhookPayloadFactory.ts

```diff
@@ -0,0 +1,89 @@
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
+
+import type { WebhookEventDTO } from "../dto/types";
+import type { IWebhookPayloadFactory } from "../interface/factory";
+import type { BookingPayloadBuilder } from "./BookingPayloadBuilder";
+import type { FormPayloadBuilder } from "./FormPayloadBuilder";
+import type { InstantMeetingBuilder } from "./InstantMeetingBuilder";
+import type { MeetingPayloadBuilder } from "./MeetingPayloadBuilder";
+import type { OOOPayloadBuilder } from "./OOOPayloadBuilder";
+import type { RecordingPayloadBuilder } from "./RecordingPayloadBuilder";
+import type { WebhookPayload } from "./types";
```

Comment:
I am not sure if we need this factory that imports all possible webhooks at once. The calling site should decide what webhook builder it needs and just import that only. It could potentially avoid 100s of imports in future or maybe now.